packages/syntax-core/src/WordConfetti/WordConfetti.tsx

```diff
@@ -50,6 +50,7 @@ const WordConfetto = ({
 }): ReactElement => {
   return (
     <Box
+      data-testid={text}
```

Comment:
@jliotta we should avoid dynamic `data-testid`'s like this

Reply:
@jliotta see https://github.com/Cambly/syntax/pull/559 for a solution - we still use the parent test id + the index